Neos.ContentRepository.Core/Classes/Feature/SubtreeTagging/Dto/SubtreeTag.php

```diff
@@ -52,6 +52,11 @@ public static function disabled(): self
         return self::instance('disabled');
     }
 
+    public static function deleted(): self
```

Comment:
if we make the idea of soft deletion a core concept then we have to explain it here i think and more importantly work out how `VisibilityConstraints::default` comes into play as the "default" should also not see deleted nodes.

Reply:
see https://github.com/neos/neos-development-collection/pull/5463#discussion_r1981198695